### DISCUSSION TITLE
New Trait/Term Discussions

### DISCUSSION THREAD

**Discussion by BoMeyering** [Planteome/CO_369-sainfoin-traits#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BoMeyering** [comment]
New traits to consider
* Leaflet count per leaf
* Legume fruit crest spike count
* Legume fruit crest spike length/height
* Legume fruit crest convexity defect
* Plant verticillium rating

**Reply by barriball** [reply]
Legume fruit yield cmp [g/St]
Stand count msr [n/Pt]

**Comment by BoMeyering** [comment]
@barriball What do you think about the traits above?

**Reply by BoMeyering** [reply]
We might also want to add leaf area and leaflet area

**Reply by barriball** [reply]
Specific Leaf Area (ratio of leaf area to leaf dry mass)

**Comment by BoMeyering** [comment]
Seedlot germination percentage [%]
* Plant 10 seeds and measure the germination rate for a seedlot.
* Expressed as a percentage

Stand count cmp [n/Pt]
* Count the total number of emerged plants in a plot or subset
* Divide by the Plot area [m2] [Ha]

Compound leaf area msr [mm2]
* Sample a compound leaf/leaves and scan at XXXdpi
* In this method, specify settings to be used with EpsonScan
    * Color balance
    * DPI
    * Window size
* Extract pixel scale from DPI and resolution
* Apply scale to image
* Threshold or mask image using segmentation modeling or routine image algorithms
* Extract total area of the compound leaf including the petiole

Compound leaf dry mass msr [mg]
* Sample on leaf and dry to constant weight in a desiccation room or in a drying oven.
* Record the dry mass in mg

Specific leaf area cmp [mm2/mg] Usually an index between compatible unit scales
* After collecting Compound leaf area [mm2] and Compound leaf dry mass msr [mg]
* Divide the area by the mass and express in mm2/mg
* Take the average if estimating multiple leaves

Leaflet area msr [mm2]
* Apply the pixel scaling factor to the image
* From a scanned image extract the leaflet contours by a segmentation model
* Extract the pixel area and mm2 area of the leaflet contour for each leaflet in the image.
* COMBINE WITH: leaflet perimeter, length, width, convexity, solidity, extent, aspect ratio, Gaborlet features??, RGB vectors....

**Comment by barriball** [comment]
Plot area msr [ha]
1. Used to convert yield traits from kg/m2 to kg/ha

Forage yield cmp [kg/ha] 

1.  used to report yield in kg/ha currently only have Forage yield cmp [kg/m2]

Legume fruit mass msr [kg]
1. Currently have Legume fruit mass msr [g] and only have Legume fruit yield cmp [kg/ha] not [g/ha]

Legume fruit mass cmp [g/ha]

**Reply by BoMeyering** [reply]
I think Plot area msr [Ha] and Forage yield cmp [kg/Ha] are fine to add in. Less convinced about the latter two. [g/Ha] doesn't seem to be the right scale to measure yield, especially over that much of an area. If anything, maybe just add in Legume fruit yield cmp [g/m2]. 

Also, right now, the mass attribute is reserved for terms that are unnormalized, so just g or kg, and not g/m2 or kg/Ha, etc

**Reply by barriball** [reply]
I think we need Legume fruit mass msr [kg]. The reason being is that if I have [g] and I have area harvested in [ha] I'd still need a conversion in the cell of g/1000 The last one should have read as Legume fruit yield cmp [g/ha] but you're right, it probably isn't necessary.

**Reply by BoMeyering** [reply]
Oh right, yes, we will need Legume fruit mass msr [kg].
